spec/globus_client_spec.rb

```diff
@@ -107,27 +107,32 @@
       end
 
       it "invokes Identity#valid?" do
-        client.user_valid?(sunetid: "bogus")
```

Comment:
huh, so this keyword arg in the spec was incorrect before?  seems like the method didn't change, and i don't see it using a kwarg in either the underlying method or the `GlobusClient` wrapper?

Reply:
IIRC, this went away a while back. Because of how we used to mock the test below, we were none the wiser. When I changed the test to use a `stub_request`-based approach, this became clear.

Reply:
ah, nice, makes sense re: mocking